docs/spec/v1alpha1/gitrepositories.md

```diff
@@ -0,0 +1,341 @@
+# Git Repositories
+
+The `GitReposiory` API defines a source for artifacts coming from Git. The
+resource exposes the latest synchronized state from Git as an artifact in
+an archive.
+
+## Specification
+
+Git repository:
+
+```go
+// GitRepositorySpec defines the desired state of a Git repository.
```

Comment:
I don't think it does define the desired state of a git repository! Maybe "... gives the specification for fetching a git repository as a source".